sdk/storage/Azure.Storage.Files.DataLake/src/Sas/DataLakeSasBuilder.cs

```diff
@@ -221,8 +221,9 @@ public DataLakeSasBuilder()
         /// class to create a Blob Service Sas.
         /// </summary>
         /// <param name="permissions">
-        /// The time at which the shared access signature becomes invalid.
-        /// This field must be omitted if it has been specified in an
+        /// The permissions associated with the shared access signature. 
```

Comment:
We need to remove the trailing whitespace so CI can pass.

Reply:
@seanmcc-msft Removed the trailing whitespaces. Could you approve [this ](https://github.com/Azure/azure-sdk-for-net/pull/26568/commits/70682e086b45b9acfc521014bca2d0e09d7ad99d) PR once you get a chance ?